src/components/MonitorLog/NonStaticQuestion.jsx

```diff
@@ -0,0 +1,292 @@
+  const [newTitleLength, setNewTitleLength] = useState(0);
+  const [newFieldType, setNewFieldType] = useState();
+  const [newTooltip, setNewTooltip] = useState();
+  const [idOfFieldBeingEdited, setIdOfFieldBeingEdited] = useState();
+  const editQuestionModal = useDisclosure();
+  const deleteQuestionModal = useDisclosure();
+
+  const updateQuestion = async () => {
+    console.log(`updateQuestion called with fieldId: ${idOfFieldBeingEdited}`);
+    console.log(`fieldBody: ${newTitle}, ${newFieldType}, ${newTooltip}`);
+    await OCHBackend.put('/forms/update/field', {
+      type: formType,
+      fieldId: idOfFieldBeingEdited,
+      fieldBody: {
+        title: newTitle,
+        subtitle: 'asdfghjkl',
+        fieldType: newFieldType,
+        tooltip: newTooltip,
+      },
+    });
+    refreshTrigger();
+    editQuestionModal.onClose();
+  };
+
+  const deleteQuestion = async () => {
+    console.log(`deleteQuestion called with fieldId ${idOfFieldBeingEdited}`);
+    await OCHBackend.delete('/forms/delete/field', {
+      data: {
+        formType,
+        fieldId: idOfFieldBeingEdited,
+      },
+    });
+    refreshTrigger();
+    deleteQuestionModal.onClose();
+  };
+
+  return (
+    <>
+      <Box
+        borderRadius="6px"
+        key={question.title}
+        _hover={isTemplate ? { bgColor: 'rgba(43, 192, 227, 0.25)' } : null}
+        onClick={
+          isTemplate
+            ? () => {
+                editQuestionModal.onOpen();
+                setNewTitle(question.title);
+                setNewTitleLength(question.title.length);
+                setNewFieldType(question.fieldType);
+                setNewTooltip(question.tooltip);
+                setIdOfFieldBeingEdited(question._id);
+                console.log(`idOfFieldBeingEdited: ${idOfFieldBeingEdited}`);
+              }
+            : null
+        }
+      >
+        <GridItem py="10px" px="5px" key={question.title} colSpan={1} rowSpan={1}>
+          <VStack spacing="8px" align="left">
+            {isTemplate ? (
+              <Text fontWeight="500" fontSize="md">
+                {question.title}
+              </Text>
+            ) : (
+              <Flex align="center">
+                <Text fontWeight="500" fontSize="md">
+                  {question.title}
+                </Text>
+                <Spacer />
+                <Tooltip label={question.tooltip} placement="top">
+                  <InfoIcon />
+                </Tooltip>
+              </Flex>
+            )}
+            {question.fieldType === 'TEXT' ? (
+              <Input type="text" />
+            ) : (
+              <NumberInput allowMouseWheel>
+                <NumberInputField />
+                <NumberInputStepper>
+                  <NumberIncrementStepper />
+                  <NumberDecrementStepper />
+                </NumberInputStepper>
+              </NumberInput>
+            )}
+            <Text color="#718096">Non-Static</Text>
```

Comment:
When displaying to volunteers, there should not be text indicating whether or not a field is non-static.